xml/templates/civicrm_state_province.tpl

```diff
@@ -4147,4 +4147,20 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (NULL, 1170, "DIN", "Dinagat Islands"),
 
 -- Add missing state for Colombia
-(NULL, 1048, "HUI", "Huila");
+(NULL, 1048, "HUI", "Huila"),
+
+-- Add missing parishes for Bermuda
+ (NULL, 'DEV', 'Devonshire'),
```

Comment:
@ginkgomzd the problem here is that your missing the country_id column (see above how it is the 2nd column)

Reply:
ah, thanks!